src/mrpro/utils/fft.py

```diff
@@ -13,37 +13,93 @@
 #   limitations under the License.
 
 import torch
+import torch.nn.functional as F
 
 
-def kspace_to_image(kdat: torch.Tensor, dim: tuple[int, ...] = (-1, -2, -3)) -> torch.Tensor:
+def change_image_shape(idat: torch.Tensor, idat_shape_new: tuple[int, ...]) -> torch.Tensor:
```

Comment:
the name "change_image_shape" somewhat suggests that this operation is for images, while we also intend to use it for k-space data.
We could also consider to make it a LinearOperator that uses `torch.nn.functional.pad`, which also works with negative paddings, making it easy to implement the adjoint as well.
the required paddings could then be inferred from the arguments `fft_recon_shape` and `fft_encoding_shape` when defining the FourierOperator